### ISSUE TITLE
Sitemap

### ISSUE THREAD

**Issue by szepeviktor** [conedevelopment/sprucecss-site-eleventy#10]
A sitemap would be useful.
https://www.npmjs.com/package/@netlify/plugin-sitemap

**Comment by adamlaki** [comment]
Yes, it would be nice and this plugin seems to be the solution. I couldn't generate one just from Eleventy easily.

Leave it open until we go live.
